package.json

```diff
@@ -64,7 +65,6 @@
   "dependencies": {
     "ajv": "^6.12.3",
     "docker-file-parser": "^1.0.7",
-    "docker-modem": "^5.0.3",
     "docker-progress": "^5.1.0",
     "dockerfile-ast": "^0.7.0",
     "dockerode": "^4.0.2",
```

Comment:
Should we also bump dockerode to [^4.0.4](https://github.com/apocas/dockerode/releases/tag/v4.0.4), which is the version that bumped docker-modem to ^5.0.6 ?
PS: fwiw the cli atm is on 4.0.5